app/src/sharedTest/java/org/oppia/app/profile/AdminAuthActivityTest.kt

```diff
@@ -132,6 +135,115 @@ class AdminAuthActivityTest {
     }
   }
 
+  @Test
+  fun testAdminAuthActivity_inputTextPreservedOnRotation() {
```

Comment:
This test case is actually testing multiple things at a time, which should not happen. Unit test cases should only check one thing at a time.

For this test case we want to check that the text entered in portrait mode still remains when shifting to landscape mode.
So the steps for that are:
1. Start activity.
2. Enter text
3. Orientation change to landscape
4. Check text is present or not.

Notice that the test case name has been updated in such a way that even the name itself should be clear about the testcase.

So the code for that would be:

```
@Test
  fun testAdminAuthActivity_inputText_configurationChange_inputTextIsPreserved() {
    launch<AdminAuthActivity>(
      AdminAuthActivity.createAdminAuthActivityIntent(
        context,
        "12345",
        -10710042
      )
    ).use {
      onView(allOf(withId(R.id.input), isDescendantOfA(withId(R.id.input_pin)))).perform(
        typeText("12345"),
        closeSoftKeyboard()
      )
      onView(isRoot()).perform(orientationLandscape())
      onView(allOf(withId(R.id.input), isDescendantOfA(withId(R.id.input_pin)))).check(matches(isDisplayed())).check(matches(withText("12345")))
    }
  }
```

Reply:
Done